scripting/shavit-zones.sp

```diff
@@ -636,6 +654,83 @@ public Action Command_Modifier(int client, int args)
 	return Plugin_Handled;
 }
 
+//Krypt Custom Spawn Functions
+
+public Action Command_AddSpawn(int client, int args)
+{
+	if(!IsValidClient(client))
+	{
+		return Plugin_Handled;
+	}
+
+	if(!IsPlayerAlive(client))
+	{
+		Shavit_PrintToChat(client, "You can't place zones when you're dead.");
+
+		return Plugin_Handled;
+	}
+
+	if(gB_CustomSpawn)
+	{
+		Shavit_PrintToChat(client, "Custom Spawn already exists. Please delete it before placing a new one.");
+
+		return Plugin_Handled;
+	}
+
+	gMZ_Type[client] = Zone_CustomSpawn;
+	GetClientAbsOrigin(client, gV_Point1[client]);
+	InsertZone(client);
+
+	return Plugin_Handled;
+}
+
+public Action Command_DelSpawn(int client, int args)
+{
+	if(!IsValidClient(client))
+	{
+		return Plugin_Handled;
+	}
+
+	char[] sQuery = new char[256];
+	FormatEx(sQuery, 256, "DELETE FROM %smapzones WHERE type = '%d' AND map = '%s';", gS_MySQLPrefix, Zone_CustomSpawn, gS_Map);
+
+	gH_SQL.Query(SQL_DeleteCustom_Spawn_Callback, sQuery, GetClientSerial(client));
+
+	return Plugin_Handled;
+}
+
+public void SQL_DeleteCustom_Spawn_Callback(Database db, DBResultSet results, const char[] error, any data)
+{
+	if(results == null)
+	{
+		LogError("Timer (custom spawn delete) SQL query failed. Reason: %s", error);
+
+		return;
+	}
+
+	int client = GetClientFromSerial(data);
+
+	if(client == 0)
+	{
+		return;
+	}
+
+	ClearCustomSpawn();
+
+	Shavit_PrintToChat(client, "Deleted Custom Spawn sucessfully.");
```

Comment:
Not a must, but you could add `gS_ChatStrings[sMessageVariable]` and `gS_ChatStrings[sMessageText]` to make this prettier :)